src/unitxt/doc_utils.py

```diff
@@ -0,0 +1,7 @@
+DOCUMENTATION_URL = "https://www.unitxt.ai/en/latest/"
+DOCUMENTATION_HUGGINGFACE_METRICS = "docs/adding_metric.html#adding-a-hugginface-metric"
+DOCUMENTATION_ADDING_TASK = "docs/adding_task.html"
```

Comment:
Maybe better in the constants in settings_utils?

Reply:
There idea is to add many such links in the error messages of unitxt, so I think a dedicated file makes sense.